src/main/java/com/github/kaklakariada/fritzbox/FritzBoxSession.java

```diff
@@ -25,75 +25,91 @@
 import com.github.kaklakariada.fritzbox.model.SessionInfo;
 
 /**
- * This class implements allows logging in to a fritzbox and execute authenticated requests.
+ * This class implements allows logging in to a fritz box and execute
+ * authenticated requests.
  */
 class FritzBoxSession {
```

Comment:
Please use eclipse formatter provided in the project, the diff is very hard to read (only mentioned once)